dotCMS/src/main/java/com/dotmarketing/business/DeterministicIdentifierAPI.java

```diff
@@ -0,0 +1,40 @@
+package com.dotmarketing.business;
+
+import com.dotcms.contenttype.model.type.ContentType;
+import com.dotmarketing.portlets.languagesmanager.model.Language;
+import java.util.function.Supplier;
+
+public interface DeterministicIdentifierAPI {
```

Comment:
I would add some doc explaining the nature of this API

Reply:
done